addon/index.ts

```diff
@@ -7,9 +7,8 @@ export {
   hasPendingWaiters,
 } from './waiter-manager';
 
-export { default as Token } from './token';
-export { default as TestWaiter } from './test-waiter';
 export { default as buildWaiter } from './build-waiter';
 export { default as waitForPromise } from './wait-for-promise';
+export { default as Token } from './token';
```

Comment:
We probably don't need to export this either, perhaps we could make a `buildToken()` helper if we really do want other folks to use the same kind of token...

Reply:
Fair. I think `Token` needs an exported type.

Reply:
Discussed offline: going to export `type Token = unknown`.